templates/_helpers.tpl

```diff
@@ -116,4 +120,11 @@ Maps object names
 {{- $path := .Values.front.env.startRoomPath }}
 {{- $universe := .Values.front.env.startRoomUniverse }}
 {{- printf "/_/%s/%s%s" $universe (include "workadventure.mapsUrl" .) $path }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{/*
+Icon object names
+*/}}
+{{- define "workadventure.icon.name" -}}
+{{- printf "%s-%s" (include "workadventure.fullname" .) .Values.icon.name | trunc 63 | trimSuffix "-" }}
+{{- end }}
```

Comment:
Where's the icon deployment? Also, this variable is never used.

Reply:
Creating the icon deployment